packages/backend/src/house/util/house.util.ts

```diff
@@ -0,0 +1,13 @@
+// Generate random house code
+// Inspired by:
+// https://stackoverflow.com/questions/1349404/generate-random-string-characters-in-javascript/1349426#1349426
+export function generateString(length) {
```

Comment:
Ideally this actually becomes an injectable class so it can be mocked for testing the `create`() call

Reply:
So just @Injectable and export this as a class?

Reply:
And probably move it into the house controller directory. Then add it to the providers in house module. Then inject it in the house controller.